src/App.js

```diff
@@ -60,9 +74,11 @@ const App = () => {
         <Route path='/' element={
           <Authorization />
         } />
-        <Route path='/main' element={
-          <Main />
-        } />
+        {localStorage.getItem("token") ?
+          <Route path='/main' element={
+            <MainPage />
+          } /> : null
```

Comment:
if you don't have token -  <Route path='/login' element={
            <loginPage />
          }